shell/platform/darwin/macos/framework/Source/FlutterView.h

```diff
@@ -4,35 +4,40 @@
 
 #import <Cocoa/Cocoa.h>
 
-/**
- * Listener for view resizing.
- */
-@protocol FlutterViewReshapeListener <NSObject>
+@protocol FlutterViewDelegate <NSObject>
 /**
  * Called when the view's backing store changes size.
  */
 - (void)viewDidReshape:(nonnull NSView*)view;
+
+- (void)scheduleOnRasterTread:(nonnull dispatch_block_t)block;
```

Comment:
typo: "RasterThread". Also, given that the embedder method is called `FlutterEnginePostRenderThreadTask`, maybe we can call this: `scheduleOnRenderThread`?

Reply:
I'm not sure about this. Apart from `FlutterEnginePostRenderThreadTask`, the thread is referred to as raster thread in most places in the code.